### PR TITLE
init code by python wechaty puppet service

### PR DESCRIPTION
**Pull request by lyleshaw**
- 根据python wechaty puppet service 项目初始化整个项目
- 修改了项目名字等地方
- 自动化部署测试等流程在后期，因此暂时删除了GitHub action的相关配置
- 修改了requirements文件

### CODE REVIEW

**Review by wj-Mcat — CHANGES_REQUESTED** [review]
Your init code is a big step for itchat puppet and it'a awesome to see your code. I have left some comments below, and I hope it will help you improve the understanding of `wechaty-puppet-*`.

File: src/wechaty_puppet_service/__init__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+from .version import VERSION
+
+
+__version__ = VERSION
+
+__all__ = [
+    'PuppetService',
+
+    '__version__'
+]
```
[the remark this answers is not in the document]
Comment (lyleshaw) [reply src/wechaty_puppet_service/__init__.py:15]: got it, I'll fix all the problems after July 3.

**Comment by lyleshaw** [comment]
I have update the code, please review it again～

File: src/wechaty_puppet_itchat/config.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import os
+# TODO: need to fix error -> found module but no type hints or library stubs
+from wechaty_puppet import get_logger   # type: ignore
+
+logger = get_logger('WechatyPuppetServiceConfig')
```
Comment (wj-Mcat) [inline src/wechaty_puppet_itchat/config.py:25]: Hi, this logger name should be `WechatyPuppetItChatConfig`.

File: src/wechaty_puppet_itchat/config.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        os.environ.get('token', None) or None
+
+
+def get_endpoint():
+    """
+    get the endpoint from environment variable
+    """
+    return os.environ.get('WECHATY_PUPPET_SERVICE_ENDPOINT', None) or \
+        os.environ.get('ENDPOINT', None) or \
+        os.environ.get('endpoint', None) or None
```
Comment (wj-Mcat) [inline src/wechaty_puppet_itchat/config.py:43]: This puppet is an implementation of `itchat`, token is not the dependency. So we can remove this config module from the repo. How do you think about it?

File: src/wechaty_puppet_itchat/puppet.py [hunk 4]
```diff
@@ -0,0 +1,1098 @@
...
+    return message_payload
+
+
+# pylint: disable=R0904
+class PuppetItChat(Puppet):
+    """
+    grpc wechaty puppet implementation
+    """
+
+    def __init__(self, options: PuppetOptions, name: str = 'puppet_service'):
```
Comment (wj-Mcat) [inline src/wechaty_puppet_itchat/puppet.py:164]: A ha, this name should be `puppet-itchat`.

Comment (wj-Mcat) [reply src/wechaty_puppet_itchat/puppet.py:164]: BTW, there are a lot `puppet service` in this module. You should read the code to refactor every method.

**Review by wj-Mcat — CHANGES_REQUESTED** [review]
I think there are also some `puppet service` you have not changed.  I hope that you can read the code more carefully.

If this step has been done, we will start the great work on communication with wechat.

File: src/wechaty_puppet_itchat/config.py (hunk 3 above)
Comment (lyleshaw) [reply src/wechaty_puppet_itchat/config.py:43]: It's appropriate

File: src/wechaty_puppet_itchat/config.py (hunk 2 above)
Comment (lyleshaw) [reply src/wechaty_puppet_itchat/config.py]: Done~

**Comment by lyleshaw** [comment]
All the problems are solved, please check the code when you have time~
Thanks.

**Review by wj-Mcat — APPROVED** [review]
LGTM!

**Pull request merged**